test/modules/aws-acm-request-integration.ts

```diff
@@ -190,6 +190,42 @@ describe('AwsAcmRequest Integration Testing', () => {
     ),
   );
 
+  it('creates a certificate request in non-default region', query(`
+      INSERT INTO certificate_request (domain_name, region)
+      VALUES ('${domainName}', 'us-east-1');
+  `));
+
+  it('applies the creation of the certificate request in non-default region', apply());
+
+  it('checks the removal of the certificate request', query(`
+      SELECT *
+      FROM certificate_request;
+  `, (res: any[]) => expect(res.length).toBe(0)));
+
+  it('checks the certificate in non-default region is created and validated', query(`
+      SELECT *
+      FROM certificate
+      WHERE domain_name = '${domainName}'
+        AND status = 'ISSUED'
+        AND region = 'us-east-1';
+  `, (res: any[]) => expect(res.length).toBe(1)));
+
+  it('deletes the certificate issued in the non-default region', query(`
+      DELETE
+      FROM certificate
+      WHERE domain_name = '${domainName}';
+  `));
+
+  it('applies the deletion of the certificate in the non-default region', apply());
+
+  it('checks the deletion of the certificate in the non-default region', query(`
+      SELECT *
+      FROM certificate
+      WHERE domain_name = '${domainName}'
+        AND status = 'ISSUED'
+        AND region = 'us-east-1';
+  `, (res: any[]) => expect(res.length).toBe(0)));
+
```

Comment:
does it make sense to have a test changing a certificate to another region?

Reply:
Changing a certificate region would be "to delete" and "to create" a certificate. Since ACM list module has nothing to do with the creation step, I think that should be done in ACM request module. The strange part is that the ACM request module removes the entry upon creation. So it should be in ACM list :))).

Basically when the `region` column in ACM list is changed, we "can" go and create an ACM certificate request for the user, but I'm not sure if it's a good idea. We might leave it to be manually done by the user. What's your own opinion on this?

Reply:
Oh, that's true. Yes, maybe it is ok as-is now and we can improve or add some feedback in the future.